packages/logger/src/formatters/test/ConsoleFormatter.test.ts

```diff
@@ -0,0 +1,114 @@
+import {ConsoleFormatter} from '../ConsoleFormatter';
+import {LogLevel} from '../..';
+
+describe('ConsoleFormatter', () => {
+  it('logs Critical entries to console.error', () => {
+    const formatter = new ConsoleFormatter();
+    const errorSpy = jest.spyOn(console, 'error');
+    const errorMsg = 'foo';
+
+    formatter.next({
+      level: LogLevel.Critical,
+      payload: new Error(errorMsg),
+      scopes: [],
+    });
+
+    expect(errorSpy).toHaveBeenCalledTimes(1);
+    const args = errorSpy.mock.calls[0];
+
+    expect(consoleCallInludes(args, errorMsg)).toBe(true);
+    expect(consoleCallInludes(args, LogLevel.Critical)).toBe(true);
+
+    errorSpy.mockReset();
+    errorSpy.mockRestore();
+  });
+
+  it('logs Critical entries using Error.message when Error.stack is null', () => {
```

Comment:
We don't seem to have a test for the case where we check the formatting when stack isn't null

Reply:
👍